src/routes/newsletters/[id].svelte

```diff
@@ -1,20 +1,89 @@
 <script lang="ts" context="module">
+  import type { LoadOutput, LoadInput } from '@sveltejs/kit';
+
   /**
    * @type {import('@sveltejs/kit').Load}
    */
-  export async function load({ fetch, page }) {
+  export async function load({ fetch, page }: LoadInput): Promise<LoadOutput> {
     const { id: slug } = page.params;
     const response = await fetch(`/newsletters/${slug}.json`);
-    return { props: { posts: await response.json() } };
+    const newsletter = await response.json();
+    if (typeof newsletter?.id !== 'number') {
+      return { status: 404 };
+    }
+    return { props: { post: newsletter } };
   }
 </script>
 
 <script lang="ts">
   import type { Newsletter } from './_query';
+  import Spacing from '$lib/components/sections/spacing.svelte';
 
-  export let posts: Newsletter[];
+  export let post: Newsletter;
 </script>
 
-<h1>Newsletter!</h1>
+<svelte:head>
+  <title>{post.title}</title>
+  <link rel="stylesheet" href="global.css" />
+</svelte:head>
+
+<Spacing --min="120px" --med="135px" --max="160px" />
+
+<section>
+  <h1>{post.title}</h1>
+
+  <Spacing --min="70px" --med="70px" --max="70px" />
+
+  <p>
+    by
+    <a href={'https://github.com/' + post.author.displayname} target="_blank">
+      @{post.author.displayname}
+    </a>
+  </p>
+
+  <Spacing --min="70px" --med="70px" --max="70px" />
+
+  <div class="container">
+    <div class="markdown-body">
+      {@html post.html}
+    </div>
+  </div>
+
+  <Spacing />
+</section>
+
+<style lang="scss">
+  @import '../node_modules/gfm.css/source/gfm.scss';

```

Comment:
Why a new dependency?

Reply:
GFM.css (GitHub-flavored markdown) is a library that mimics the styles used for READMEs on github.com. The idea was that we'd use GFM so the newsletters would look similar on both GitHub and on acmcsuf.com/newsletters. GitHub has a [_Preview_ feature](https://github.blog/changelog/2021-11-17-preview-the-markdown-rendering-of-gists/) which is also helpful for drafting our newsletters.